setup.cfg

```diff
@@ -70,10 +70,12 @@ tests =
     pytest-cov>=2.5.1
     pre-commit
     matplotlib>=3.4.2
+    IPython
```

Comment:
I couldn't see what IPython was being used for?

Reply:
This was mistakenly included. There was an issue with vis_cpu importing IPython, but not actually including it properly in its install dependencies. I have now made it optional in this PR: https://github.com/HERA-Team/vis_cpu/pull/13